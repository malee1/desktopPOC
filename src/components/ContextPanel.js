import { React, useEffect } from 'react';
import { makeStyles } from '@material-ui/core/styles';
import { useDispatch, useSelector } from 'react-redux';
import axios from 'axios';
import Logo from './Logo';
import PanelOne from './PanelOne';
import PanelTwo from './PanelTwo';
import PanelThree from './PanelThree';
import PanelOneAccess from './PanelOneAccess';
// import PanelAccess from './PanelAccess';
import { getCustomerApi, getUserApi } from '../constants/api';
import { getCustomer, getUser } from '../redux/actions';
import user from '../constants/user';

const useStyles = makeStyles({
  root: {
    height: '20%',
    display: 'flex',
    overflow: 'hidden',
    padding: '5px'
  },
  logoContainer: {
    minWidth: '15vw',
    display: 'flex',
    marginTop: '15px'
  },
  panelContainer: {
    flexGrow: 10,
    display: 'flex'
  }
});

export default function ContextPanel() {
  const classes = useStyles();

  const customerUrl = getCustomerApi;
  const userUrl = getUserApi;

  const showAccessibilityPanel = useSelector((state) => state.accessibility.accessibility);

  const dispatch = useDispatch();

  useEffect(() => {
    const getCustomerData = async () => {
      try {
        const response = await axios(customerUrl);
        dispatch(getCustomer(response.data));
      } catch (error) {
        console.log(`Error getting customer data: ${error}`);
      }
    };
    getCustomerData();
  }, [customerUrl, dispatch]);

  useEffect(() => {
    const getUserData = async () => {
      try {
        const response = await axios.post(userUrl, user);
        dispatch(getUser(response.data));
      } catch (error) {
        console.log(`Error getting user data: ${error}`);
      }
    };
    getUserData();
  }, [userUrl, dispatch]);

  return (
    <div className={classes.root} data-testid="context-panel">
      <div className={classes.logoContainer}>
        <Logo className={classes.logo} data-testid="logo" />
      </div>
      <div className={classes.panelContainer} data-testid="panel-container">
        {/* <PanelOne className={classes.panel} /> */}
        {/* <PanelOneAccess /> */}
        {showAccessibilityPanel ? <PanelOneAccess /> : <PanelOne />}
        <PanelTwo className={classes.panel} />
        <PanelThree className={classes.panel} />
      </div>
    </div>
  );
}
