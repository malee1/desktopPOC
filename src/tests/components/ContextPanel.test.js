import React from 'react';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import renderer from 'react-test-renderer';
import { cleanup, render } from '@testing-library/react';

import ContextPanel from '../../components/ContextPanel';

const mockStore = configureMockStore();
const store = mockStore({
  customer: {
    data: {
      name: 'Test Test',
      dob: '1111',
      uRef: '2222',
      tel: '3333',
      timeWait: 'aaaa',
      context: 'bbbb',
      recording: 'cccc',
      callInfo: 'dddd',
      alerts: 'Important Info 1',
      service: 'Alert1',
      keyInfo: 'Info 1',
      auth: 'Approved'
    }
  },
  user: {
    data: {
      fileId: '123456',
      roleProfile: 'Alpha',
      boltOn: ''
    }
  },
  accessibility: {
    accessibility: false
  }
});

afterEach(() => {
  cleanup();
  jest.clearAllMocks();
});

describe('the panel renders', () => {
  it('matches the snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <ContextPanel />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

describe('it renders the components correctly for default users', () => {
  it('it renders the whole component', () => {
    const { getAllByTestId } = render(
      <Provider store={store}>
        <ContextPanel />
      </Provider>
    );
    const container = getAllByTestId('context-panel');
    expect(container.length).toBe(1);
  });
  it('it renders the logo component', () => {
    const { getAllByTestId } = render(
      <Provider store={store}>
        <ContextPanel />
      </Provider>
    );
    const container = getAllByTestId('logo');
    expect(container.length).toBe(1);
  });
  it('it renders the panel container component', () => {
    const { getAllByTestId } = render(
      <Provider store={store}>
        <ContextPanel />
      </Provider>
    );
    const container = getAllByTestId('panel-container');
    expect(container.length).toBe(1);
  });
  it('it renders the 3 panels', () => {
    const { getAllByTestId } = render(
      <Provider store={store}>
        <ContextPanel />
      </Provider>
    );
    const container1 = getAllByTestId('panel-one');
    expect(container1.length).toBe(1);
    const container2 = getAllByTestId('panel-two');
    expect(container2.length).toBe(1);
    const container3 = getAllByTestId('panel-three');
    expect(container3.length).toBe(1);
  });
});

const store2 = mockStore({
  customer: {
    data: {
      name: 'Test Test',
      dob: '1111',
      uRef: '2222',
      tel: '3333',
      timeWait: 'aaaa',
      context: 'bbbb',
      recording: 'cccc',
      callInfo: 'dddd',
      alerts: 'Important Info 1',
      service: 'Alert1',
      keyInfo: 'Info 1',
      auth: 'Approved'
    }
  },
  user: {
    data: {
      fileId: '123456',
      roleProfile: 'Alpha',
      boltOn: 'Accessibility'
    }
  },
  accessibility: {
    accessibility: true
  }
});

describe('it renders the components correctly for users with accessibility needs', () => {
  it('it renders the whole component', () => {
    const { getAllByTestId } = render(
      <Provider store={store2}>
        <ContextPanel />
      </Provider>
    );
    const container = getAllByTestId('context-panel');
    expect(container.length).toBe(1);
  });

  it('it renders the correct panel 1', () => {
    const { getAllByTestId } = render(
      <Provider store={store2}>
        <ContextPanel />
      </Provider>
    );
    const container1 = getAllByTestId('panel-one-access');
    expect(container1.length).toBe(1);
  });
});
