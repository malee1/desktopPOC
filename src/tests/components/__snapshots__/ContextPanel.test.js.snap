// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the panel renders matches the snapshot 1`] = `
<div
  className="makeStyles-root-1"
  data-testid="context-panel"
>
  <div
    className="makeStyles-logoContainer-2"
  >
    <div
      className="makeStyles-root-4"
      data-testid="logo"
    >
      <img
        alt="logotest"
        className="makeStyles-image-5"
        src="/logo192.png"
      />
    </div>
  </div>
  <div
    className="makeStyles-panelContainer-3"
    data-testid="panel-container"
  >
    <div
      className="makeStyles-root-6"
      data-testid="panel-one"
    >
      <div
        className="MuiBox-root MuiBox-root-19 makeStyles-box-7"
      >
        <ul
          className="MuiList-root makeStyles-list-8 MuiList-padding"
        >
          <li
            className="MuiListItem-root makeStyles-listItem-9 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-11"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-12"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-13"
            >
              <p
                className="MuiTypography-root makeStyles-label-14 MuiTypography-body1"
                data-testid="aaa"
              >
                Name: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-15"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-16 MuiTypography-body1"
              >
                Test Test
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItem-9 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-11"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-12"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-13"
            >
              <p
                className="MuiTypography-root makeStyles-label-14 MuiTypography-body1"
              >
                Date of Birth: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-15"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-16 MuiTypography-body1"
              >
                1111
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItem-9 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-11"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-12"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M17.81 4.47c-.08 0-.16-.02-.23-.06C15.66 3.42 14 3 12.01 3c-1.98 0-3.86.47-5.57 1.41-.24.13-.54.04-.68-.2-.13-.24-.04-.55.2-.68C7.82 2.52 9.86 2 12.01 2c2.13 0 3.99.47 6.03 1.52.25.13.34.43.21.67-.09.18-.26.28-.44.28zM3.5 9.72c-.1 0-.2-.03-.29-.09-.23-.16-.28-.47-.12-.7.99-1.4 2.25-2.5 3.75-3.27C9.98 4.04 14 4.03 17.15 5.65c1.5.77 2.76 1.86 3.75 3.25.16.22.11.54-.12.7-.23.16-.54.11-.7-.12-.9-1.26-2.04-2.25-3.39-2.94-2.87-1.47-6.54-1.47-9.4.01-1.36.7-2.5 1.7-3.4 2.96-.08.14-.23.21-.39.21zm6.25 12.07c-.13 0-.26-.05-.35-.15-.87-.87-1.34-1.43-2.01-2.64-.69-1.23-1.05-2.73-1.05-4.34 0-2.97 2.54-5.39 5.66-5.39s5.66 2.42 5.66 5.39c0 .28-.22.5-.5.5s-.5-.22-.5-.5c0-2.42-2.09-4.39-4.66-4.39-2.57 0-4.66 1.97-4.66 4.39 0 1.44.32 2.77.93 3.85.64 1.15 1.08 1.64 1.85 2.42.19.2.19.51 0 .71-.11.1-.24.15-.37.15zm7.17-1.85c-1.19 0-2.24-.3-3.1-.89-1.49-1.01-2.38-2.65-2.38-4.39 0-.28.22-.5.5-.5s.5.22.5.5c0 1.41.72 2.74 1.94 3.56.71.48 1.54.71 2.54.71.24 0 .64-.03 1.04-.1.27-.05.53.13.58.41.05.27-.13.53-.41.58-.57.11-1.07.12-1.21.12zM14.91 22c-.04 0-.09-.01-.13-.02-1.59-.44-2.63-1.03-3.72-2.1-1.4-1.39-2.17-3.24-2.17-5.22 0-1.62 1.38-2.94 3.08-2.94 1.7 0 3.08 1.32 3.08 2.94 0 1.07.93 1.94 2.08 1.94s2.08-.87 2.08-1.94c0-3.77-3.25-6.83-7.25-6.83-2.84 0-5.44 1.58-6.61 4.03-.39.81-.59 1.76-.59 2.8 0 .78.07 2.01.67 3.61.1.26-.03.55-.29.64-.26.1-.55-.04-.64-.29-.49-1.31-.73-2.61-.73-3.96 0-1.2.23-2.29.68-3.24 1.33-2.79 4.28-4.6 7.51-4.6 4.55 0 8.25 3.51 8.25 7.83 0 1.62-1.38 2.94-3.08 2.94s-3.08-1.32-3.08-2.94c0-1.07-.93-1.94-2.08-1.94s-2.08.87-2.08 1.94c0 1.71.66 3.31 1.87 4.51.95.94 1.86 1.46 3.27 1.85.27.07.42.35.35.61-.05.23-.26.38-.47.38z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-13"
            >
              <p
                className="MuiTypography-root makeStyles-label-14 MuiTypography-body1"
              >
                Unique Reference: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-15"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-16 MuiTypography-body1"
              >
                2222
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItem-9 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-11"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-12"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M6.62 10.79c1.44 2.83 3.76 5.14 6.59 6.59l2.2-2.2c.27-.27.67-.36 1.02-.24 1.12.37 2.33.57 3.57.57.55 0 1 .45 1 1V20c0 .55-.45 1-1 1-9.39 0-17-7.61-17-17 0-.55.45-1 1-1h3.5c.55 0 1 .45 1 1 0 1.25.2 2.45.57 3.57.11.35.03.74-.25 1.02l-2.2 2.2z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-13"
            >
              <p
                className="MuiTypography-root makeStyles-label-14 MuiTypography-body1"
              >
                Tel: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-15"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-16 MuiTypography-body1"
              >
                3333
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItemButton-10 MuiListItem-gutters"
            disabled={false}
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-panelButton-18 MuiButton-containedSecondary"
              data-testid="button"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                <p
                  className="MuiTypography-root MuiTypography-body2"
                >
                  More Info
                </p>
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
              </span>
            </button>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="makeStyles-root-20"
      data-testid="panel-two"
    >
      <div
        className="MuiBox-root MuiBox-root-33 makeStyles-box-21"
      >
        <ul
          className="MuiList-root makeStyles-list-22 MuiList-padding"
        >
          <li
            className="MuiListItem-root makeStyles-listItem-23 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-25"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-26"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                />
                <path
                  d="M12.5 7H11v6l5.25 3.15.75-1.23-4.5-2.67z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-27"
            >
              <p
                className="MuiTypography-root makeStyles-label-28 MuiTypography-body1"
              >
                Time Waiting: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-29"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-30 MuiTypography-body1"
              >
                aaaa
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItem-23 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-25"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-26"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm1 17h-2v-2h2v2zm2.07-7.75l-.9.92C13.45 12.9 13 13.5 13 15h-2v-.5c0-1.1.45-2.1 1.17-2.83l1.24-1.26c.37-.36.59-.86.59-1.41 0-1.1-.9-2-2-2s-2 .9-2 2H8c0-2.21 1.79-4 4-4s4 1.79 4 4c0 .88-.36 1.68-.93 2.25z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-27"
            >
              <p
                className="MuiTypography-root makeStyles-label-28 MuiTypography-body1"
              >
                Context: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-29"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-30 MuiTypography-body1"
              >
                bbbb
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItem-23 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-25"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-26"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 14c1.66 0 2.99-1.34 2.99-3L15 5c0-1.66-1.34-3-3-3S9 3.34 9 5v6c0 1.66 1.34 3 3 3zm5.3-3c0 3-2.54 5.1-5.3 5.1S6.7 14 6.7 11H5c0 3.41 2.72 6.23 6 6.72V21h2v-3.28c3.28-.48 6-3.3 6-6.72h-1.7z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-27"
            >
              <p
                className="MuiTypography-root makeStyles-label-28 MuiTypography-body1"
              >
                Recording: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-29"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-30 MuiTypography-body1"
              >
                cccc
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItem-23 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-25"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-26"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <circle
                  cx="9"
                  cy="9"
                  r="4"
                />
                <path
                  d="M9 15c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4zm7.76-9.64l-1.68 1.69c.84 1.18.84 2.71 0 3.89l1.68 1.69c2.02-2.02 2.02-5.07 0-7.27zM20.07 2l-1.63 1.63c2.77 3.02 2.77 7.56 0 10.74L20.07 16c3.9-3.89 3.91-9.95 0-14z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-27"
            >
              <p
                className="MuiTypography-root makeStyles-label-28 MuiTypography-body1"
              >
                Call Info: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-29"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-30 MuiTypography-body1"
              >
                dddd
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItemButton-24 MuiListItem-gutters"
            disabled={false}
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-panelButton-32 MuiButton-containedSecondary"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                More Context
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
              </span>
            </button>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="makeStyles-root-34"
      data-testid="panel-three"
    >
      <div
        className="MuiBox-root MuiBox-root-47 makeStyles-box-35"
      >
        <ul
          className="MuiList-root makeStyles-list-36 MuiList-padding"
        >
          <li
            className="MuiListItem-root makeStyles-listItem-37 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-39"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-40"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 22c1.1 0 2-.9 2-2h-4c0 1.1.89 2 2 2zm6-6v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-.83-.67-1.5-1.5-1.5s-1.5.67-1.5 1.5v.68C7.63 5.36 6 7.92 6 11v5l-2 2v1h16v-1l-2-2z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-41"
            >
              <p
                className="MuiTypography-root makeStyles-label-42 MuiTypography-body1"
              >
                Customer Alerts: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-43"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-44 MuiTypography-body1"
              >
                Important Info 1
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItem-37 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-39"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-40"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M20 2H4c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-7 9h-2V5h2v6zm0 4h-2v-2h2v2z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-41"
            >
              <p
                className="MuiTypography-root makeStyles-label-42 MuiTypography-body1"
              >
                Service Alerts: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-43"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-44 MuiTypography-body1"
              >
                Alert1
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItem-37 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-39"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-40"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M3 18h12v-2H3v2zM3 6v2h18V6H3zm0 7h18v-2H3v2z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-41"
            >
              <p
                className="MuiTypography-root makeStyles-label-42 MuiTypography-body1"
              >
                Key Info: 
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-43"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-44 MuiTypography-body1"
              >
                Info 1
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItem-37 MuiListItem-gutters"
            data-testid="item"
            disabled={false}
          >
            <div
              className="makeStyles-iconContainer-39"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root makeStyles-icon-40"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.11 0 2-.9 2-2V5c0-1.1-.89-2-2-2zm-9 14l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"
                />
              </svg>
            </div>
            <div
              className="makeStyles-labelContainer-41"
            >
              <p
                className="MuiTypography-root makeStyles-label-42 MuiTypography-body1"
              >
                Authorisation
              </p>
            </div>
            <div
              className="makeStyles-labelValueContainer-43"
            >
              <p
                className="MuiTypography-root makeStyles-labelValue-44 MuiTypography-body1"
              >
                Approved
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root makeStyles-listItemButton-38 MuiListItem-gutters"
            disabled={false}
          >
            <button
              className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-panelButton-46 MuiButton-containedSecondary"
              disabled={false}
              onBlur={[Function]}
              onClick={[Function]}
              onDragLeave={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={0}
              type="button"
            >
              <span
                className="MuiButton-label"
              >
                More Alert Details
                <svg
                  aria-hidden={true}
                  className="MuiSvgIcon-root"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
              </span>
            </button>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
